cf-serverd/server_tls.c

```diff
@@ -184,7 +184,7 @@ void ServerTLSDeInitialize()
  *
  * @return -1 in case of error, 1 otherwise.
  */
-int ServerTLSPeek(ConnectionInfo *conn_info)
+bool ServerTLSPeek(ConnectionInfo *conn_info)
```

Comment:
Don't forget to adjust the comments in all the functions you change the signature.

Reply:
Thanks, fixed.